driver/build/Dockerfile

```diff
@@ -21,7 +22,9 @@ LABEL name="IBM Spectrum Scale CSI driver" \
       run='docker run ibm-spectrum-scale-csi-driver' \
       summary="An implementation of CSI Plugin for the IBM Spectrum Scale product."\
       description="CSI Plugin for IBM Spectrum Scale"\
-      maintainers="IBM Spectrum Scale"
+      maintainers="IBM Spectrum Scale"\
```

Comment:
Not your change, but the line above, should we sync that up with the "summary" similar to the operator description and summary are the same.

Reply:
Yeah I think  summary and description should be linked, let me push  a pr to that effect.